python/ray/tune/ray_trial_executor.py

```diff
@@ -189,7 +189,6 @@ def __init__(self,
         self._cached_actor_pg = deque(maxlen=1)
 
         self._avail_resources = Resources(cpu=0, gpu=0)
```

Comment:
Should this be staying?

Reply:
Seems like it's used in `debug_string`, which is printed out  in Tune loop. I find the information quite useful.
How about let's keep it here for now.